be/src/util/json_converter.cpp

```diff
@@ -37,8 +38,10 @@ class SimdJsonConverter {
             return JsonValue(builder.slice());
         } catch (simdjson::simdjson_error& e) {
             std::string_view view(value.get_raw_json_string().raw());
-            auto err_msg = strings::Substitute("Failed to convert simdjson value, json=$0, error=$1", view.data(),
-                                               simdjson::error_message(e.error()));
+            // truncate the raw json string if it is too large
+            size_t size = view.size() > MAX_VALUE_LENGTH_FOR_ERRMSG ? MAX_VALUE_LENGTH_FOR_ERRMSG : view.size();
```

Comment:
usually append addition error messages if is truncated, e.g. `'original_json_string ... <truncated> `